mimic/rest/nova_api.py

```diff
@@ -399,6 +413,44 @@ def perform_action(self, request, tenant_id, server_id):
         """
         return self._region_collection_for_tenant(tenant_id).request_action(request, server_id, self.url)
 
+    @app.route("/v2/<string:tenant_id>/os-keypairs", methods=['GET'])
+    def get_key_pairs(self, request, tenant_id):
+        """
+        Returns current key pairs
+        """
+        return self._keypair_collection_for_tenant(tenant_id).json_list()
+
+    @app.route("/v2/<string:tenant_id>/os-keypairs", methods=['POST'])
+    def create_key_pair(self, request, tenant_id):
+        """
+        Returns a newly created key pair with the specified name.
+        """
+        try:
+            content = json.loads(request.content.read())
+            keypair = content["keypair"]
+            keypair_from_request = KeyPair(name=keypair["name"], public_key=keypair["public_key"])
```

Comment:
Ok, sounds good. Can you please add a issue for this, if you do not intend on working on this immediately